Sources/tart/VM.swift

```diff
@@ -278,6 +282,18 @@ class VM: NSObject, VZVirtualMachineDelegate, ObservableObject {
     // Entropy
     configuration.entropyDevices = [VZVirtioEntropyDeviceConfiguration()]
 
+    // Directory share
+    if #available(macOS 13, *) {
```

Comment:
What do you think throwing an error on Monterey?

Reply:
See https://github.com/cirruslabs/tart/pull/211/commits/01884f7021edb87b6a193c973704dd60950c7c95.